.github/workflows/clippy-check.yml

```diff
@@ -1,26 +1,23 @@
 on: [pull_request]
 name: Clippy
 jobs:
-  clippy_check:
+  format:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - uses: actions-rs/toolchain@v1
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@master
         with:
-          components: clippy, rustfmt
+          components: rustfmt
           toolchain: nightly
-          override: true
-      - uses: actions-rs/cargo@v1
-        with:
-          command: fmt
-          args: --all -- --check
-      - name: Install cargo-lints
-        uses: actions-rs/cargo@v1
-        with:
-          command: install
-          args: cargo-lints
-      - name: Clippy lints
-        uses: actions-rs/cargo@v1
+      - name: Formatting
+        run: cargo +nightly fmt --all -- --check
+  clippy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@master
         with:
-          command: lints
-          args: clippy --all-targets
\ No newline at end of file
+          components: clippy
+          toolchain: stable
+      - name: Lints
+        run: cargo clippy --all-targets
```

Comment:
this is not correct. The full command is 
`cargo lints clippy --all-targets` so that it uses the lints.toml file

Reply:
Odd that `cargo clippy` won't use the lint file...

Reply:
As far as I have it we need to run, or this is what I always run locally:
```
cargo clippy --all-targets --all-features -- -D warnings
```
and
```
cargo lints clippy --all-targets --all-features
```
Those are two separate checks.

Reply:
I added the `clippy::all` group to the lints file as a deny, which should capture this behavior.